gcp/.env

```diff
@@ -0,0 +1,25 @@
+softhsm_helm_version=12.0.1
+softhsm_docker_image=mosipid/softhsm
```

Comment:
@ckm007 

> Lets just add chart version in the .env, docker image and version is not needed.
This is keeping in mind that the releases charts always contains itsrelevant configurations like docker image, tag etc.

in some of the helm charts, qa docker images are used, hence we gave an option to override it,
and this is completely optionally and can be skipped.

> Also, lets add source chart version as a configuration so that in case of development mode someone can host the helm charts in their repo and use the same.

Sure, we will provide an option to pass the helm chart repository from .env file

Reply:
Fixed it by providing an option  in .env file to update the helm repo

Reply:
By default lets keep the image and tag parameters commented. 

Add an instruction which says uncomment in case want to deploy docker image different from the one packaged with the helm chart.